From 2368a976be94becdac216b4a09919010fd78b7cb Mon Sep 17 00:00:00 2001
From: Andrew Miyaguchi <XDleader555@gmail.com>
Date: Fri, 28 Jul 2017 16:34:20 -0700
Subject: [PATCH] arm/dts: msm8992: Add clock-frequency property

This commit will remove the following error message at boot

[0.062725,0] /cpus/cpu@0: Missing clock-frequency property
[0.062741,0] /cpus/cpu@1: Missing clock-frequency property
[0.062754,0] /cpus/cpu@2: Missing clock-frequency property
[0.062768,0] /cpus/cpu@3: Missing clock-frequency property
[0.062783,0] /cpus/cpu@4: Missing clock-frequency property
[0.062798,0] /cpus/cpu@5: Missing clock-frequency property
---
 arch/arm/boot/dts/qcom/msm8992.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/msm8992.dtsi b/arch/arm/boot/dts/qcom/msm8992.dtsi
index 38998c1b538..da44d9c5c61 100644
--- a/arch/arm/boot/dts/qcom/msm8992.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8992.dtsi
@@ -75,6 +75,7 @@
 		CPU0: cpu@0 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
+			clock-frequency = <1440000000>;
 			reg = <0x0>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc0>;
@@ -110,6 +111,7 @@
 		CPU1: cpu@1 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
+			clock-frequency = <1440000000>;
 			reg = <0x1>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc1>;
@@ -139,6 +141,7 @@
 		CPU2: cpu@2 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
+			clock-frequency = <1440000000>;
 			reg = <0x2>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc2>;
@@ -168,6 +171,7 @@
 		CPU3: cpu@3 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
+			clock-frequency = <1440000000>;
 			reg = <0x3>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc3>;
@@ -197,6 +201,7 @@
 		CPU4: cpu@4 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a57";
+			clock-frequency = <1824000000>;
 			reg = <0x100>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc4>;
@@ -236,6 +241,7 @@
 		CPU5: cpu@5 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a57";
+			clock-frequency = <1824000000>;
 			reg = <0x101>;
 			enable-method = "qcom,8994-arm-cortex-acc";
 			qcom,acc = <&acc5>;
